app/app/settings.py

```diff
@@ -137,6 +145,8 @@
     'social_core.backends.github.GithubOAuth2',
     'social_core.backends.azuread_tenant.AzureADTenantOAuth2',
     'django.contrib.auth.backends.ModelBackend',
+    'django.contrib.auth.backends.RemoteUserBackend' # added for custom SSO
```

Comment:
Nit: missing trailing comma. (Trailing commas make future git diffs clearer since adding a new entry to the end of the array only needs to touch the added line.)